composer.json

```diff
@@ -65,10 +65,17 @@
         "jms/serializer-bundle": "^2.0 || ^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.1.1",
         "nelmio/api-doc-bundle": "^2.4",
+        "phpstan/extension-installer": "^1.0",
+        "phpstan/phpstan": "^1.0",
+        "phpstan/phpstan-phpunit": "^1.0",
+        "phpstan/phpstan-symfony": "^1.0",
```

Comment:
phpstan strict rules too ?

Reply:
I would like to but they generate too many errors, probably is safer to do it on next major instead to avoid bugs on stable releases.